.env.e2e.example

```
@@ -71,7 +71,7 @@ MAX_CREDITS_BONUS=1500
 # To trigger credits assignment probe need to be online >20 hours during the day. Current "Probes status CRON" is "*/10 * * * *", which is 6 times an hour
 # So required value is 20 * 6 = 120
 ADOPTED_PROBES_REQUIRED_ONLINE_TIMES=120
-TARGET_NODE_VERSION=v20.13.0 # Same format as in gp_probes.nodeVersion. Make sure to update the triggers in a new migration.
+TARGET_NODE_VERSION=v22.16.0 # Same format as in gp_probes.nodeVersion. Make sure to update the triggers in a new migration.
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)